infrastructure/tooling/src/smoketest/checks/createClient.js

```diff
@@ -0,0 +1,21 @@
+const taskcluster = require('taskcluster-client');
+
+exports.tasks = [];
+
+exports.tasks.push({
+  title: 'Create client and use it (--target create-client)',
+  requires: [],
+  provides: [
+    'create-client',
+  ],
+  run: async () => {
+    const auth = new taskcluster.Auth(taskcluster.fromEnvVars());
+    const randomId = taskcluster.slugid();
+    let clientId = `project/taskcluster/smoketest/${randomId}`;
+    const payload = {
+      "expires": taskcluster.fromNowJSON('1 hour'),
+      "description": `Create a client and use it ${clientId}`,
```

Comment:
You will probably want to add some scopes here, so that you can use the resulting client to do something. I'd suggest allowing the client to reset its own access token, so `auth:reset-access-token:project/taskcluster/smoketest/${randomId}`.

Reply:
Awesome!